packages/wonder-blocks-dropdown/src/components/action-item.js

```diff
@@ -38,6 +38,12 @@ type ActionProps = {|
      */
     href?: string,
 
+    /**
+     * Optional attribute to indicate to the Screen Reader which language the
+     * item text is in.
+     */
+    lang?: string,
```

Comment:
Are there any other places that we should be adding this prop?  If so, maybe file a JIRA to take care of the rest in the new year.

Reply:
Not required yet, but better to tackle this in advance for sure!